frontend/src/components/Login/index.jsx

```diff
@@ -1,13 +1,54 @@
+import { useState } from "react";
+import axios from "axios";
 import Style from "./style";
 
 export default function Login() {
+  const [formData, setFormData] = useState({
+    email: "",
+    password: "",
+  });
+
+  const hChange = (evt) => {
+    const { name, value } = evt.target;
+    setFormData({ ...formData, [name]: value });
+  };
+
+  const hSubmit = (evt) => {
+    evt.preventDefault();
+    axios
+      .post(`${import.meta.env.VITE_BACKEND_URL}/auth/login`, formData)
+      .then(() => {
+        console.warn("welcome");
+      })
+      .catch(() => {
+        console.error("wrong login");
```

Comment:
Even this is not really useful, you could instead dispatch a Toast notification ?